pkg/notify/notify.go

```diff
@@ -172,67 +173,68 @@ func toIssueLabel(groupLabels alertmanager.KV) string {
 	return strings.Replace(buf.String(), " ", "", -1)
 }
 
-func (r *Receiver) search(project, issueLabel string) (*jira.Issue, bool, error) {
+func (r *Receiver) search(project, issueLabel string, logger log.Logger) (*jira.Issue, bool, error) {
 	query := fmt.Sprintf("project=\"%s\" and labels=%q order by resolutiondate desc", project, issueLabel)
 	options := &jira.SearchOptions{
 		Fields:     []string{"summary", "status", "resolution", "resolutiondate"},
 		MaxResults: 2,
 	}
-	log.V(1).Infof("search: query=%v options=%+v", query, options)
+	level.Debug(logger).Log("msg", "searching for existing issue", "query", query, "options", options)
```

Comment:
Please leave these log messages as they were (minus the formatted parameters). They are only intended for debugging, go after one another in the log and are easier to eyeball as they are:
```
msg="Search" query="XXX"
msg="  found" issue="YYY"
```
or
```
msg="Create" issue="XXX"
msg="  done" key="YYY"
```

Reply:
Personally, I'm not sure I like this extra formatting when talking about logs. Nevertheless, I've change all occurrences back as requested.

Reply:
You seem to have left out the indentation, though. That's the thing that allows you to be brief in all but the first message.

Reply:
Could you point to the exact lines where indentation is still missing? I thought I've added it (two extra spaces) to the places I've removed it initially. It's possible I've missed some places though.

Reply:
Seems like indentation is all OK now.